extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/URIExtractionNamespaceFunctionFactory.java

```diff
@@ -151,7 +151,7 @@ public String call()
             versionRegex = null;
           }
         } else {
-          final Path filePath = Paths.get(extractionNamespace.getUri());
+          final Path filePath = Paths.get(extractionNamespace.getUri().getPath());
```

Comment:
This is just going to be the path part (not the scheme or the authority), is this going to do the right thing? In particular uriBase is not going to have the same scheme as the original uri.